dApp-Svr-Exp/modules/ballots/ballots-list-ctrl.js

```diff
@@ -82,15 +43,17 @@
       function getOption(votes, pos) {
         const totalVoted = getTotalVotes(votes);
         const desc = votes.options[pos-1];
-        const total = getPorcent(totalVoted, votes.totals[pos-1]);
```

Comment:
Typos from here on: Percentage, not porcent. Can we fix them now before it spreads more?